docs/intermediate/tutorial11-normals/README.md

```diff
@@ -151,12 +151,13 @@ Some model formats include the tanget and bitangent (sometimes called the binorm
 
 ![](./tangent_space.png)
 
-Basically we can use the edges of our triangles, and our normal to calculate the tangent and bitangent. But first, we need to update our `ModelVertex` struct in `model.rs`.
+Basically we can use the edges of our triangles, and our normal to calculate the tangent and bitangent. But first, we need to update our `ModelVertex` struct in `model.rs`. In addition to adding two new fields `tangent` and `bitangent`, we also need to change the types from `[f32, N]` to `cgmath::VectorN<f32>` so that we can do some matrix calculations later.
 
 ```rust
 #[repr(C)]
 #[derive(Copy, Clone, Debug)]
 pub struct ModelVertex {
+    // UPDATED!
```

Comment:
Here you could add a comment explaining why the change to use `cgmath` was made (ie. to simplify the tangent, bitanget calculation).